x-pack/plugins/elastic_assistant/server/lib/langchain/graphs/default_assistant_graph/index.ts

```diff
@@ -121,10 +130,11 @@ export const callAssistantGraph: AgentExecutor<true | false> = async ({
     dataClients,
     llm,
     logger,
-    messages,
     tools,
+    responseLanguage,
+    replacements,
   });
-  const inputs = { input: latestMessage[0].content as string };
+  const inputs = { input: latestMessage[0]?.content as string };
 
   if (isStream) {
     return streamGraph({ apmTracer, assistantGraph, inputs, logger, onLlmResponse, request });
```

Comment:
FYI, @patrykkopycinski and I noticed LangSmith tracing wasn't working with streaming. Turns out I forgot to pass in `traceOptions` here which includes the necessary tracer. It's plumbed through the rest of the way, so only need to add it to this call.

Patryk is going to fix this in his esql PR, so need to add it now if you don't want, but wanted to let you know in case you were looking for traces it in LangSmith.